### ISSUE TITLE
Powershell right-arrow complete on historical entries broken after update to 2020.6.0

### ISSUE THREAD

**Issue by jlapchuk** [PowerShell/vscode-powershell#2773]
### System Details

System Details Output
```
### VSCode version: 1.46.0 a5d1cc28bb5da32ec67e86cc50f84c67cc690321 x64

### VSCode extensions:
ms-vscode.powershell@2020.6.0
VisualStudioExptTeam.vscodeintellicode@1.2.8


### PSES version: 2.2.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.18362.752
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, 5.0, 5.1.18362.752}
BuildVersion                   10.0.18362.752
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

### Issue Description

I am experiencing a problem with...

Right-arrow complete of historical PowerShell commands not showing up

#### Expected Behaviour

While typing out a command, PowerShell should suggest a command line that was previously typed in, allowing me to hit right-arrow to complete the line and save me from retyping. 

#### Actual Behaviour

The suggestions don't show up

### Attached Logs

[EditorServices.log](https://github.com/PowerShell/vscode-powershell/files/4794275/EditorServices.log)
[StartEditorServices.log](https://github.com/PowerShell/vscode-powershell/files/4794276/StartEditorServices.log)
[vscode-powershell.log](https://github.com/PowerShell/vscode-powershell/files/4794277/vscode-powershell.log)

**Comment by jlapchuk** [comment]
I can also confirm that downgrading to v2020.4.0 of the PowerShell Extension resolves the problem. So there's some sort of breaking change in the latest version.

**Comment by corbob** [comment]
This is due to the updated PSReadline that turns that off by default. https://twitter.com/Steve_MSFT/status/1269073033941008384 for more information.

**Comment by jlapchuk** [comment]
Ok, thanks for the info. I downloaded the pre-release version of PSReadline 
[https://github.com/PowerShell/PSReadLine/releases/tag/v2.1.0-beta2](url)

Added the extra "Set-PSReadLineOption -PredictionSource History" to my profile... All is good now :)

**Comment by jlapchuk** [comment]
Ok, thanks for the info. I downloaded the pre-release version of PSReadline 
[https://github.com/PowerShell/PSReadLine/releases/tag/v2.1.0-beta2](url)

Added the extra "Set-PSReadLineOption -PredictionSource History" to my profile... All is good now :)

**Comment by TylerLeonhardt** [comment]
Yes we accidentally shipped PSRL preview with 2020.4.0... only stable builds should ship with stable PowerShell extensions and doing exactly that is how you can opt in to the preview :)
